### PR TITLE
@duplicate: added option to reset acl

### PR DESCRIPTION
**Pull request by inakip**

### CODE REVIEW

File: guillotina/content.py [hunk 1]
```diff
@@ -694,26 +696,36 @@ def get_all_behavior_interfaces(content) -> list:
...
         if key.startswith("__") or key.startswith("_BaseObject"):
             continue
-        if key in ("id",):
+        if key in ("id", "creators", "contributors"):
             continue
         new_obj.__dict__[key] = context.__dict__[key]
+
     new_obj.__acl__ = context.__acl__
+    if reset_acl:
+        prinrole = IPrincipalRoleManager(context)
```
Comment (vangheem) [inline guillotina/content.py]: Should use the IGetOwner:

```
get_owner = get_utility(IGetOwner)
        roleperm = IPrincipalRoleManager(new_obj)
        owner = await get_owner(new_obj, user)
        if owner is not None:
            roleperm.assign_role_to_principal("guillotina.Owner", owner)
```

File: guillotina/content.py [hunk 2]
```diff
@@ -694,26 +696,36 @@ def get_all_behavior_interfaces(content) -> list:
...
-
     for key in context.__dict__.keys():
         if key.startswith("__") or key.startswith("_BaseObject"):
             continue
-        if key in ("id",):
+        if key in ("id", "creators", "contributors"):
             continue
         new_obj.__dict__[key] = context.__dict__[key]
+
     new_obj.__acl__ = context.__acl__
```
Comment (vangheem) [inline guillotina/content.py]: Should be in else?

File: guillotina/content.py [hunk 3]
```diff
@@ -694,26 +696,36 @@ def get_all_behavior_interfaces(content) -> list:
...
+    if reset_acl:
+        prinrole = IPrincipalRoleManager(context)
+        prinrole.map._clear()
```
Comment (vangheem) [inline guillotina/content.py]: Should do `new_obj.__acl__ = None` instead!

File: guillotina/content.py (hunk 1 above)
Comment (inakip) [reply guillotina/content.py]: ok, thanks

File: guillotina/content.py (hunk 3 above)
Comment (inakip) [reply guillotina/content.py]: ok, thanks

**Comment by vangheem** [comment]
@bloodbare @masipcat Are you okay with this change? Can we merge?

@inakip please also make change to 5.x!

**Pull request merged**